glide.lock

```diff
@@ -28,7 +28,9 @@ imports:
 - name: github.com/kardianos/osext
   version: c2c54e542fb797ad986b31721e1baedf214ca413
 - name: github.com/mailru/easyjson
-  version: 9d6630dc8c577b56cb9687a9cf9e8578aca7298a
+  version: c7e8f2b687054e448f35ff2693bd298c1a2f3a10
+  repo: https://github.com/Raynos/easyjson.git
```

Comment:
If you pin a specific version of easyjson, then  maybe always point to the master version of the customized repo, because then git repo will sync only with new stable version.